src/main/java/com/squareup/javawriter/ParameterWriter.java

```diff
@@ -18,15 +18,24 @@
 import java.io.IOException;
 
 public final class ParameterWriter extends VariableWriter {
+  private boolean isVararg;
+
   ParameterWriter(TypeName type, String name) {
     super(type, name);
   }
 
+  public void setIsVararg(boolean isVararg) {
```

Comment:
I think I still like `setVarargs` best. In normal code you'd never actually set varargs on anything (e.g., reflective method invocation) because it's always just an array. Your setter would be semantic in name (e.g., setColumns). Varargs only exist in code, and we're an API for writing code. Does it still bug you after that?